storage/txcache/txListBySenderMap.go

```diff
@@ -54,7 +53,7 @@ func (txMap *txListBySenderMap) getListForSender(sender string) (*txListForSende
 }
 
 func (txMap *txListBySenderMap) addSender(sender string) *txListForSender {
-	listForSender := newTxListForSender(sender, txMap.cacheConfig)
+	listForSender := newTxListForSender(sender, &txMap.cacheConfig)
```

Comment:
:+1: this is fine as the only structure able to change the values from the config is txListBySenderMap. The caller of `newTxListBySenderMap` will not be able to change the configs provided to the txListBySenderMap instance.